### PR TITLE
Add ShellLauncher v2 information for CSP

### PR DESCRIPTION
**Pull request by chandde**

### CODE REVIEW

**Comment by e0i** [comment]
@lomayor A PR that has been created by a MS contributor is available for review. Thanks.

**Comment by chandde** [comment]
@lomayor  @DulceMontemayor  can you help take a look at this change? thanks!

**Comment by chandde** [comment]
@lomayor @DulceMontemayor ping...

**Comment by lomayor** [comment]
@chandde , this topic has been assigned to me in bulk. I can't help validate content but I can approve the PR. Can you confirm the content is accurate?

**Comment by chandde** [comment]
Hi @lomayor , I can confirm the content change is accurate

**Comment by lomayor** [comment]
Thanks, @chandde! Can you resolve the conflicts first?

**Comment by chandde** [comment]
@lomayor  Done!

File: windows/client-management/mdm/assignedaccess-csp.md [hunk 1]
```diff
@@ -992,6 +1000,31 @@ Status Get
     </xs:element>
 </xs:schema>
 ```
+### Shell Launcher V2 XSD
+
+``` syntax
```
Comment (illfated) [inline windows/client-management/mdm/assignedaccess-csp.md]: Please use **```xml** for correct XML syntax highlighting.

File: windows/client-management/mdm/assignedaccess-csp.md [hunk 2]
```diff
@@ -1112,6 +1145,61 @@ ShellLauncherConfiguration Add AutoLogon
 </SyncML>
 ```
 
+ShellLauncher V2 Add
+```
```
Comment (illfated) [inline windows/client-management/mdm/assignedaccess-csp.md]: Suggestion: **```xml**

File: windows/client-management/mdm/assignedaccess-csp.md (hunk 1 above)
Comment (illfated) [reply windows/client-management/mdm/assignedaccess-csp.md]: Sorry, I forgot that this was a merged PR.

File: windows/client-management/mdm/assignedaccess-csp.md (hunk 2 above)
Comment (illfated) [reply windows/client-management/mdm/assignedaccess-csp.md]: Sorry, I forgot that this was a merged PR.

**Pull request merged**